src/styles/blocks/top-row-layout.scss

```diff
@@ -0,0 +1,33 @@
+.top-row-layout {
+
+  &--one-icon {
+    @include headerNavigationGrid(1, 18px, 30px);
+    @include onTablet {
+      @include headerNavigationGrid(1, 24px, 63px);
+    }
+  }
+
+  &--two-icons {
+    @include headerNavigationGrid(2, 18px, 30px);
+    @include onTablet {
+      @include headerNavigationGrid(2, 24px, 63px);
+    }
+  }
+
+  &__link-logo {
+    cursor: auto;
```

Comment:
The 'cursor: auto;' property is not necessary for the link logo, as this is the default cursor value for non-link elements. You can safely remove this line.